### DISCUSSION TITLE
cant upload anything bcs of cor error

### DISCUSSION THREAD

**Discussion by sadeq1375** [supabase/supabase#22945] [Questions]
i cant upload song or image bcs of this errors:
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://ysgccobgoglbbxbfitac.supabase.co/storage/v1/object/songs/song-vaghti%20raftam-lv7m0d0k. (Reason: CORS request did not succeed). Status code: (null).

statusCode | "400"
error | "Error"
message | "headers must have required property 'authorization'"

**Comment by GaryAustin1** [comment]
You would need to provide more info on how you are uploading.  The error seems to be you don't have an authorization header which is required for upload.
